etc/distrepos.conf

```diff
@@ -72,7 +73,8 @@ previous_root = /data/repo.previous
 logfile = /var/log/distrepos.log
 
 # The base location of mirrorlist files.
-mirror_root = /data/mirror/osg
+mirror_root = /data/mirror
```

Comment:
Will this result in the [same layout we currently have](https://repo.opensciencegrid.org/mirror/osg/)? Where does the "osg" get added?

Reply:
Currently every tagset config has "osg" at the start of its dest field so it's not needed in the `_root` fields, eg:

```ini
[tagset osg-23-main-$${EL}-release]
dvers = el8 el9
dest = osg/23-main/$${EL}/release
```

It might make sense to remove it from the individual tagsets and pull it up to the root though